Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1569,6 +1569,21 @@ public class BrowserViewController: UIViewController {
     }
   }
 
+  func showSNSDomainInterstitialPage(originalURL: URL, visitType: VisitType?) {
```

Comment:
Please Pass the VisitType as unknown as default parameter and get of the if let check

Reply:
updated in [a4cd1a8](https://github.com/brave/brave-ios/pull/6906/commits/a4cd1a858eb2db471ba0d554b8a31267b8cf01fa)